package.json

```diff
@@ -8,6 +8,7 @@
   "devDependencies": {
     "@commitlint/cli": "^17.6.7",
     "@commitlint/config-conventional": "^17.6.7",
+    "@lavamoat/allow-scripts": "^3.0.0",
```

Comment:
this is not necessary, because `@lavamoat/allow-scripts` is in the `packages/allow-scripts` directory.

```suggestion
```